src/testParser.ts

```diff
@@ -285,6 +295,7 @@ async function parseSuite(
 
         annotations.push({
           path: resolvedPath,
+          testcase: testcase._attributes.name,
```

Comment:
could you please elaborate on the need for adding the testcase to the annotations?

Based on the API for annotations there does not seems to be the property testcase included:

https://docs.github.com/en/rest/reference/checks